src/status_im/ui/screens/routing/profile_stack.cljs

```diff
@@ -17,6 +17,7 @@
                      :dapps-permissions
                      :manage-dapps-permissions
                      :extensions-settings
+                     :extension-screen-holder
```

Comment:
don't we want to have it in wallet stack as well?

Reply:
right